package.json

```
@@ -55,6 +55,7 @@
     "build:cjs": "rollup -c rollup.cjs.config.js",
     "dev": "rollup -c -w",
     "test": "jest test/index.spec.ts --runInBand",
+    "test:validation": "jest test/validation.spec.ts --runInBand",
```

It would be better to have a build script before running the test to be sure it use latest version of the validation code, like this:
```suggestion
    "test:validation": "npm run build:cjs && jest test/validation.spec.ts --runInBand",
```